software/docker/kafka-can-log/kafka_can_log/kafka_can_log.c

```diff
@@ -0,0 +1,415 @@
+/*
```

Comment:
Side note: This might be a fun service to try and do a rust re-write on.

Reply:
Given your expert skills in Rust, I think you can do it. :)